.codecov.yaml

```diff
@@ -0,0 +1,3 @@
+ignore:
+  - "cmd/sidetree-server"  # ignore folders and all its contents
```

Comment:
why do we need to ignore?

Reply:
It is main.go. It will be tested using integration tests.